pkg/datapath/loader/netlink.go

```diff
@@ -329,3 +332,57 @@ func SetupBaseDevice(mtu int) (netlink.Link, netlink.Link, error) {
 
 	return linkHost, linkNet, nil
 }
+
+// reloadIPSecOnLinkChanges subscribes to link changes to detect newly added devices
+// and reinitializes IPsec on changes. Only in effect for ENI mode in which we expect
+// new devices at runtime.
+func (l *Loader) reloadIPSecOnLinkChanges() {
+	// settleDuration is the amount of time to wait for further link updates
+	// before proceeding with reinitialization. This avoids reinitializing
+	// twice when seeing NEWLINK followed by SETLINK.
+	const settleDuration = 5 * time.Second
+
+	if !option.Config.EnableIPSec || option.Config.IPAM != ipamOption.IPAMENI {
+		return
+	}
+
+	updates := make(chan netlink.LinkUpdate, 16)
```

Comment:
Where is the 16 coming from?

Reply:
The buffering isn't strictly necessary so could just drop that.